src/__test__/reducers/signupreducer.test.js

```diff
@@ -0,0 +1,60 @@
+import { isTSAnyKeyword, exportAllDeclaration } from '@babel/types';
+import signupReducer from '../../reducers/signupReducer';
+import { registerUser, registerUserSucess, registerUserfailure } from '../../actions/signupAction';
```

Comment:
This too, it seems to be not used at all. kindly take it out.

Reply:
@NicholusMuwonge this has been worked on